scripts/justmat/foulplay.lua

```diff
@@ -65,7 +72,9 @@ for i=1, 8 do
   track[i].n = 16
   track[i].pos = 1
   track[i].s = {}
-  track[i].prob = 100                                                         
+  track[i].prob = 100 
+  track[i].trig_logic = 0              -- 0 == none, 1 == and, 2 == or
+  track[i].logic_target = track_edit   -- trig logic track target
```

Comment:
just FWIW, you might consider a record-style initialization:

```lua

track[i] = {
  k = 0,
  n = 16,
  pos = 1,
  ...
}
```

a little more terse.  your call though!

Reply:
Thanks for the detailed review. Still very new to this, so I appreciate the advice! This does look much more clean.

Reply:
no problem.  we're all just learning.  it's fun!